### ISSUE TITLE
Syntax error on version 1.1.0

### ISSUE THREAD

**Issue by titospeakap** [drevops/behat-format-progress-fail#24]
I have been using version 1.0.0 without any problem, but when upgrading to version 1.1.0, I get the error 

```
syntax error, unexpected 'ResultToStringConverter' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)  
```

In the behat.yaml file, I have 

```
default:
    extensions:
        FriendsOfBehat\SymfonyExtension:
            kernel:
                environment: test
                debug: false
                class: Speakap\AppKernel
        Bex\Behat\StepTimeLoggerExtension:
            enabled_always: false
            output: [csv]
        DrevOps\BehatFormatProgressFail\FormatExtension: ~
    formatters:
        progress_fail: ~
        pretty: false
```

any suggestion on how to fix it?

**Comment by AlexSkrypnyk** [comment]
@titospeakap 
Thank you for reporting this. I will look into this as a priority.

**Comment by AlexSkrypnyk** [comment]
Is there more information about where in code this happened?

**Comment by AlexSkrypnyk** [comment]
@titospeakap 
I was not able to replicate your issue.

But I've notices that you have used `progress_fail: ~`, while it should be `progress_fail: true`

**Comment by titospeakap** [comment]
thanks for looking into this @AlexSkrypnyk 

the error that I get is the one described above and it happens on file 

```
| In PrinterProgressFail.php line 29:
|                                                                                                                      
|   syntax error, unexpected 'ResultToStringConverter' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)  
```

I also changed to `progress_fail: true` but same error.. btw, I'm on PHP 7.2

**Comment by AlexSkrypnyk** [comment]
@titospeakap 
This is caused by PHP version. The typed class properties are supported only starting from 7.4

The version 1.1.0 of this package is compatible with ^8.1 only, but this constraint is not specified in ‘composer.json’, which is a bug.

once fixed, you will not be able to use 1.1.0 until you upgrade your php version. But at least composer will not be installing an incompatible version.

**Comment by titospeakap** [comment]
once again using an outdated version of PHP bit us.. for now we will lock the version... thanks @AlexSkrypnyk for looking into this.. I will close this issue..
